change/@fluentui-react-drawer-6eaa1e97-38f8-4d08-8748-8eecd859a585.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32816&commitId=c7ae6a3487ef94ed121e809fdbc9b3e60fed9418&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|
|Avatar Converged.badgeMask.chromium.png|17| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1139| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2219| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>